testing/tests/DevExpress.core/devices.tests.js

```diff
@@ -50,21 +46,7 @@ QUnit.test('ios by userAgent', function(assert) {
 });
 
 QUnit.test('android by userAgent', function(assert) {
-    let device = fromUA(userAgents.android_4_3_4);
-
-    assert.equal(device.platform, 'android', 'platform is android');
-    assert.equal(device.version.toString(), '4,3,4', 'correct version');
-    assert.equal(device.deviceType, 'phone', 'deviceType is phone');
-    assert.equal(device.grade, 'B', 'grade is B');
-
-    device = fromUA(userAgents.android_4_4_0);
-
-    assert.equal(device.platform, 'android', 'platform is android');
-    assert.equal(device.version.toString(), '4,4,0', 'correct version');
-    assert.equal(device.deviceType, 'phone', 'deviceType is phone');
-    assert.equal(device.grade, 'A', 'grade is A');
-
-    device = fromUA(userAgents.android_tablet_7_1_1);
+    let device = fromUA(userAgents.android_tablet_7_1_1);
```

Comment:
```suggestion
    const device = fromUA(userAgents.android_tablet_7_1_1);
```

Reply:
It need be `let`, cause we change this variable below.

Reply:
Can you show me the line where the `device` variable is reassigned?